source/_drafts/videojs-vr-now-under-the-video-js-org.md

```diff
@@ -0,0 +1,57 @@
+---
+title: videojs-vr Is Now Under The Video.js Org
+author:
+  name: Brandon Casey
+  github: brandonocasey
+tags: vr, 360, plugins
+---
+
+Brightcove and Video.js have adopted the videojs-vr plugin as a first class plugin.
+It's out now with support for Video.js 6!
+
+## Where did it come from?
+This plugin has a long history and has had many maintainers.
+It started as [slawrence/videojs-vr](https://github.com/slawrence/videojs-vr). From there it became [metacdn/videojs-vr](https://github.com/metacdn/videojs-vr). Then there was a private collaberation between [StreamShark.io](https://streamshark.io), [Brightcove](https://www.brightcove.com), and [HapYak](http://corp.hapyak.com/). Eventually we decided to treat this plugin like a first class citizen. So we I updated it to the [advanced plugin API](http://docs.videojs.com/tutorial-plugins.html#writing-an-advanced-plugin), fixed a few long-standing issues, documented other issues along the way, and moved it to the Video.js Organization.
```

Comment:
`So we I updated` should be `So we updated`

Reply:
collaberation -> collaboration